### DISCUSSION TITLE
custom field in kimai2 and mobile

### DISCUSSION THREAD

**Discussion by acopernic** [kimai/Paid-plugins#52] [General]
Hi,
I may be interested in the custom field plugin but...
> Which version is compatible. I see the v2 is not (even if I don't understand what is the v2 - I have the version 1.30.1 from git)
> On the Kimai mobile can we "see" these custom fields and if yes which version - BTW the Kimai mobile shows all the activities even without any client/project selected
Thanks. 
PS: Bravo for the project and best wishes for this new year.

**Comment by kevinpapst** [comment]
You are on the latest version. This is compatible!

v2 is the next major version coming within the next weeks. 
See the demo page: https://www.kimai.org/demo/

If you mean the mobile app, then: I don't know. It is made by a different company. Please ask them.

**Comment by schwantuschke** [comment]
Is custom field ready for Kimai v2? If not, when can it be expected?
